tests/test_marc.py

```diff
@@ -158,12 +185,179 @@ def test_marc_record_all_fields_transform_correctly():
+            ),
+            timdex.Note(
+                value=["Lamoureux Concerts Orchestra ; Igor Markevitch, conductor"],
+                kind="Participant or Performer Note",
+            ),
+            timdex.Note(
+                value=["Berlin Symphony Orchestra ; Kurt Sanderling, conductor"],
+                kind="Participant or Performer Note",
+            ),
+            timdex.Note(
+                value=["Suspended publication 1944-52"],
+                kind="Numbering Peculiarities Note",
+            ),
+            timdex.Note(
+                value=["Some numbers combined"], kind="Numbering Peculiarities Note"
+            ),
+            timdex.Note(value=["Canada"], kind="Geographic Coverage Note"),
+            timdex.Note(value=["Mexico"], kind="Geographic Coverage Note"),
+            timdex.Note(
+                value=[
+                    "Electronic reproduction. New York : Springer, 2008. Mode of access: "
+                    "World Wide Web. System requirements: Web browser. Title from title "
+                    "screen (viewed on June 27, 2008). Access may be restricted to users "
+                    "at subscribing institutions"
+                ],
+                kind="Reproduction Note",
+            ),
+            timdex.Note(
+                value=[
+                    "Microfiche. Washington : U.S. Govt. Print. Off., 1981. 1 microfiche "
+                    "; 11 x 15 cm"
+                ],
+                kind="Reproduction Note",
+            ),
+            timdex.Note(
+                value=[
+                    "First published in United States New York : Ballantine Books, an "
+                    "imprint of Random House, a division of Penguin Random House, 2021"
+                ],
+                kind="Original Version Note",
+            ),
+            timdex.Note(
+                value=[
+                    "Originally published New York : Garland, 1987. Series statement 1 "
+                    "Series statement 2"
+                ],
+                kind="Original Version Note",
+            ),
+            timdex.Note(
+                value=["Hard copy version record"],
+                kind="Source of Description Note",
+            ),
+            timdex.Note(
+                value=["Paper copy version record"],
+                kind="Source of Description Note",
+            ),
+            timdex.Note(
+                value=["Rare Book copy: Advance copy notice inserted"],
+                kind="Local Note",
+            ),
+            timdex.Note(value=["Advance copy notice inserted"], kind="Local Note"),
+        ],
         numbering="-Bd. 148, 4 (dez. 1997). Began in 1902.",
         physical_description=(
             "484 p. : ill. ; 30 cm. + 1 CD-ROM (4 3/4 in.). 1 DVD-ROM "
             "(4 3/4 in.). 1 vocal score (248 p.) ; 31 cm."
         ),
         publication_frequency=["Six no. a year", "Three times a year"],
+        related_items=[
```

Comment:
It seems very hard to make all the `related_item.description` attributes "pretty" so I opted for minimizing excess punctuation, let me know what you think

Reply:
We need to review all of these related item subfields anyway so I just made a new ticket for that in the backlog. This is fine for now.